### ISSUE TITLE
Batch Endpoint - AutoML

### ISSUE THREAD

**Issue by MurugeswariMuthurajan** [Azure/mlops-v2#49]
Hi,

We opted for automated ML , ran the job and registered the model in dev workspace
Now we want to deploy it in test and prod workspace. We have downloaded the .pkl file, scoring script, conda env file which are all required for deployment. 

![image](https://user-images.githubusercontent.com/107349920/185043830-6e1f0bfb-c959-4415-89da-15c493717d7d.png)

But it seems the scoring script generated is applicable only for online deployment. So whether batch deployment is not possible in automated ml?

Thanks

**Comment by setuc** [comment]
Yes its possible for AutoML too. The steps remain the same just like for any Batch deployment. 

You can refer to the pattern here: https://github.com/Azure/mlops-project-template/blob/main/classical/aml-cli-v2/mlops/devops-pipelines/deploy-batch-endpoint-pipeline.yml

In addition, we have started work on multiple workspaces and this is going to be part of our next release sometime in November 2022.
